### PR TITLE
Add docs for column count tag

### PR DESCRIPTION
**Pull request by mattgreenfield**
Pretty self explanatory really.

Maybe this is too much for a glossary page? It will be much nicer in the side by side docs thing rather than a tutorial I think. Leaving it here for reference? Deleting it all together? I'm impartial...

@AidanThreadgold 

P.S I haven't tested this with `jekyll serve` because its broken on my computer....

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ layout: index
 title: Change Log
 ---
 
+## 30th March 2015 - Column Count Tag
+Add documentation for `<!--WDK:column:count-->`, required for the default styling of the new re-sizeable category layout.
```
Comment (AidanThreadgold) [inline changelog.md]: "grid category layout" ?

**Comment by AidanThreadgold** [comment]
Looks good man!

File: changelog.md (hunk 1 above)
Comment (mattgreenfield) [inline changelog.md]: yer, why not ey! I think we call it tile, but grid is better....

**Comment by mattgreenfield** [comment]
Cheers

**Comment by Dayjo** [comment]
Do we still force it to 2 on the design screen? Or was this removed as part of design studio and removal of drag and drop in place columns?

**Comment by AidanThreadgold** [comment]
We don't force it on the Design Screen anymore, no - but I think the drag/drop JS is still there when it shouldn't be.

**Pull request merged**